plugin.py

```diff
@@ -76,8 +81,7 @@ def onStart(self):
 
         # create the mandatory child device if it does not yet exist
         if 1 not in Devices:
-            Domoticz.Device(Name="Status", Unit=1, TypeName="Custom",Options={"Custom": "1;Phase"},Used=1).Create()
-
+            Domoticz.Device(Name="Status", Unit=1, TypeName="Custom",Options={"Custom": "1;Days"},Used=1).Create()
```

Comment:
Nice addition. I 've renamed on my installed device but didn't had the chance to update the code. Nice!!